lib/jsxgettext.js

```diff
@@ -186,8 +191,14 @@ function parse(sources, options) {
             }
           };
           if( arg.constructor === Array ) {
-              translations[str].msgid_plural = extractStr(arg[1]);
-              translations[str].msgstr = ['', ''];
+              //If this is context string
+              if (arg.length === 3) {
+                  translations[str].msgctxt = extractStr(arg[1]);
+              //this is plural
+              } else {
+                translations[str].msgid_plural = extractStr(arg[1]);
+                translations[str].msgstr = ['', ''];
+              }
```

Comment:
It wont work well, since it makes impossible to use plurals with context and all other future combinations.

Though, I must say that to make it work original section must be refactored too.